docs/source/getting-started/install.md

```diff
@@ -219,7 +219,7 @@ Use the project generator helper utility.
 
     ```bash
     $ npm install -g yo @plone/generator-volto
-    $ yo @plone/volto
+    $ yo @plone/volto --canary
```

Comment:
On line 216, I would change this:

```
Use the project generator helper utility.
The latest stable release of Volto will be installed by default.
You may choose to install the canary version, which is the latest alpha release, using the `--canary` flag.
```

Then for this line:

```suggestion
    # install latest stable release
    yo @plone/volto
    # or install latest alpha release
    yo @plone/volto --canary
```

Reply:
This is not completely resolved. Please include the content on Line 216 as well. Then it will be complete. Thank you!

Reply:
Done!